Pinta.Tools/Tools/TextTool.cs

```diff
@@ -433,7 +434,8 @@ protected override void OnMouseDown (Document document, ToolMouseEventArgs e)
 			Point pt = e.PointDouble.ToGdkPoint ();
 
 			// Grab focus so we can get keystrokes
-			document.Workspace.Canvas.GrabFocus ();
+			//PintaCore.Chrome.Canvas.GrabFocus();
```

Comment:
Please remove this line rather than commenting it out

Reply:
ok, I will change soon.